packages/dendron-cli/src/commands/publishCLICommand.ts

```diff
@@ -414,14 +408,39 @@ export class PublishCLICommand extends CLICommand<CommandOpts, CommandOutput> {
     });
   }
 
-  async _initialize(opts: { nextPath: string; spinner: ora.Ora }) {
-    const { spinner } = opts;
+  /**
+   * Clone template
+   * If templateVersion is set in dendron.yml, clone at version
+   * Otherwise, get same version as engine
+   * @param opts
+   * @return: string, version of template
+   */
+  async _initialize(opts: {
```

Comment:
since this only seems to clone a template, and its return value is a templateVersion, should we just rename this method 'cloneTemplate'?

Reply:
it does a few things outside of cloning so would keep the current name, especially since we already have a `this._cloneTemplate` method

- read config to get templateVersion
- clone template
- switch to correct version of template based on `templateVersion`
- install depdenencies
- write `templateVersion` to `dendron.yml` if not set